pkg/client-go/kubevirt/clientset/versioned/clientset.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright 2023 The KubevirtApiLifecycleAutomation Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
we should probably update the year here to 2024, and also I didn't see any licence so do we even need that ?

Reply:
I think we can modify it to 2025 at this point :)
Done